modules/core/shared/src/main/scala/lucuma/core/enum/Band.scala

```diff
@@ -48,18 +48,18 @@ sealed abstract class Band(
 }
 
 trait BandDefaultUnit {
-  trait DefaultUnit[B <: Band, G <: BrightnessUnit.Group] {
-    val unit: GroupedUnitType[G]
+  trait DefaultUnit[N <: Band, B <: BrightnessUnit.Group] {
```

Comment:
Kind of like the previous type names `B` for brightness and `G` for group

Reply:
I started using `B` for brightness and then when I got here it clashed with band's `B` :D

Anyway, my next step is trying to remove `BrightnessUnit.Group` to simplify things (`BrightnessUnit.Integrated` and `BrightnessUnit.Surface` would just be some unrelated phantom types), but I'm not sure that's sanely possible.

I'll go back to using `B` for `Band` in any case.